Updates to 12.6 changelog. (#33188)
Automattic/jetpack: projects/plugins/jetpack/CHANGELOG.md

## 12.6 - 2023-09-19
### Enhancements
- Add blogroll item block. [#32746]
- Added the new auto-conversion toggle for social. [#32597]
- Add number formatting for subscriber count in newsletter module. [#32646]

> - Add wpcom_newsletter_categories_enabled site option. [#32569]

Reviewer: These should be redone in the usual "Feature: add whatever. [#xxxxx]" style, then sorted into where they belong.

- AI Excerpt: add the Discard button to the panel. [#32890]
- AI Excerpt: ensure to save the post before asking a suggestion. [#32891]
- AI Excerpt: show upgrade banner when site achieves requests limit. [#32895]
- AI Extension: add create-with-voice beta extension. [#32742]
- AI Extension: enhance blocks parsing flickering with child blocks. [#32635]
- AI Extension: introduce Create with voice jetpack block - initial implementation. [#32750]
- AI Extension: use new AIControl component from AI package and remove local implementation. [#32682]
- Business Hours Block: refactor block registration to avoid duplication. [#32698]
- Classic Editor Markdown: add support for double-tilde strikethrough, e.g. `~~strikethrough~~`. [#33057]
- Connection UI: use Redux state to pass the "from" property. [#32600]
- Content Lens: connect excerpt panel with AI Assistant. [#32846]
- Create with voice: connect block with useMediaRecording() hook. [#32769]
- Create with voice: introduce Done button. [#32792]
- Create with voice: set block properties by using useBlockProps() custom hook. [#32782]
- Create with voice: show audio duration when recording audio. [#32827]

> - Fix a potential bug. [#32768]

Reviewer: Either clarify this (what bug, in what?), or just remove the line.

- Gravatar profile widget: add srcset. [#32662]
- Jetpack Assistant: add placeholder UX for the prompt input. [#33080]
- Jetpack Social: display the Social editor panel with an invitation to activate the feature when it is disabled. [#33149]
- Launchpad save modal: persist the user option when they opt to hide the modal. [#32567]

> - Link styles and scripts to block.json automatically. [#32720]

Reviewer: Feature prefix?

- Newsletter: add support for newsletter categories. [#32666]

> - Newsletters: improve flows around access and paywall. [#32615]

Reviewer: Both should be either "Newsletter" or "Newsletters". Probably the latter I think?

> - Nicer subscribers number. [#32651]

Reviewer: Feature prefix?

- Paywall: better content for when waiting for email validation. [#32800]
- Paywall Block: add toolbar options. [#32659]
- Paywall block: disable Paid subscribers button if plans are not set up. [#32663]
- Paywall block: improved settings panel. [#32801]
- Paywall Block: improved toolbar. [#32765]
- Paywall Block: improve excerpts handling. [#32809]
- Paywall block: redesign visuals for the mobile editor. [#32672]

> - Paywall block: release to WP.com and Jetpack. [#32625]

Reviewer: Be consistent with "block" vs "Block". But I'm not sure which we prefer.

Also line 47 seems kind of pointless to have in here.

- Related Posts: make the block available even when the feature has not been activated yet. [#32930]

> - Render Subscriptions block on emails. [#32409]

Reviewer: ```suggestion
- Subscriptions block: render in emails. [#32409]
```

- Subscriptions: remove newsletter flow+theme and FSE theme check for the subscribe modal. Leave WP.com check. [#32607]
- Tiled gallery module: add srcset to images. [#32598]
- WordPress.com Toolbar: add site monitoring menu and remove site logs. [#32458]

### Improved compatibility
- Carousel: add new hook allowing third-parties to customize the metadata displayed within the Carousel modal. [#32749]
- General: Jetpack now requires WordPress version 6.2. [#32762]
- General: remove WP 6.1 backwards compatibility checks. [#32772]
- Lazy Images: discourage enabling of Lazy loading images as now 90% of browser support the feature natively and this version in Jetpack can conflict with the upcoming WordPress interactivity API. [#33030]
- Shortcodes: exclude images with broken URL from galleries. [#32556]
- Subscriptions: change the text color of selected newsletter categories to be more easily readable. [#33104]
- Theme fields: only add is_block_theme to theme response when WordPress version is < 6.3 [#31660]
- WC Analytics: ensure correct values are tracked for cart/checkout block/template tracking when using WC Blocks cart and checkout templates. [#32715]

### Bug fixes

> - Adding test for default whitelist option. [#32632]

Reviewer: Feature prefix? If we even need to mention adding a test?

- Admin Menu: display Newsletter settings submenu item for Jetpack connected sites. [#33065]
- AI Chat: fix block styles not loading. [#33090]
- AI Chat: fix styles not loading in the editor. [#33120]
- AI Extension: remove HTML fragments from AI-generated forms on site editor. [#32798]

> - Allow two year plans to show on the My Plan page. [#32904]

Reviewer: ```suggestion
- My Plans: allow two year plans to show on the page. [#32904]
```

- Blocks: use custom Facebook and Instagram icons in Jetpack's Facebook and Instagram blocks to avoid issues in accessibility inspector. [#32705]
- Comments: remove mention of Twitter as a log in option since it is no longer available. [#32717]
- Comments: remove the Twitter service as a login provider. [#30273]
- Donation Block: fix issue with Danish Krone. [#33072]

> - Email subscribers now receive full posts. [#32647]

Reviewer: Feature prefix?

- Featured images fallback: fix undefined variable warnings. [#33179]

> - Fix an issue that caused the notice displayed when updating widgets to overlap the admin menu. [#32921]

Reviewer: ```suggestion
- Widgets: fix an issue that caused the notice displayed when updating widgets to overlap the admin menu. [#32921]
```
I think?

- Fix block icons for display on wp.org. [#32754]
- Fix cookie consent block positioning in the editor. [#32622]
- Fix default plan being selected in payment block. [#32640]

> - Fixed a Global Styles UI performance issue in the Site Editor. [#32641]

Reviewer: More things needing feature prefixes.

- Launchpad modal: reduce API calls for modal dismiss flag. [#32751]

> - Load the Jetpack block assets in the iframe directly in WP-Admin and keep the existing behavior on the front-end, for compatibility reasons. [#32706]

Reviewer: Feature prefix?

- Nav Unification: flips order of tags and categories submenu of posts. [#32619]
- Paywall block: avoid undefined variable warning. [#32578]
- Paywall Block: fix autosave before redirecting the user. [#33033]

> - Remove use of Jetpack_Membership::get_connected_account_id(). [#32620]

Reviewer: ```suggestion
- Remove use of `Jetpack_Membership::get_connected_account_id()`. [#32620]
```
Also, what feature? Or "General"?

> - Revert change where newsletter plans could not be used on Premium content. [#32850]

Reviewer: ```suggestion
- Newsletters: allow Newsletter plans in the Premium Content and Recurring Payment blocks again. [#32850]
```

## 12.6 - 2023-09-19
### Enhancements
- AI Assistant: add prompts for correct feature configuration. [#33064]

> - Classic Editor Markdown: add support for double-tilde strikethrough, e.g. ~~strikethrough~~. [#33057]

Reviewer: Possibly we should quote the example, otherwise it renders as "~~strikethrough~~" in the changelog itself. 😀
```suggestion
- Classic Editor Markdown: add support for double-tilde strikethrough, e.g. `~~strikethrough~~`. [#33057]
```

- Jetpack Assistant: add placeholder UX for the prompt input. [#33080]
- Jetpack Social: display the Social editor panel with an invitation to activate the feature when it is disabled. [#33149]
- Business Hours Block: refactor block registration to avoid duplication. [#32698]
- Related Posts: make the block available even when the feature has not been activated yet. [#32930]
- Security: Blocks: avoid saving invalid block attributes that may appear to other editors on the site. [#33184]
- Sidebar: Rename the "Inbox" menu to "My Mailboxes". [#32992]
- Site Editor: point the Site Editor's `<` link to the theme showcase when previewing a theme. [#32913]
- Subscription block: add newsletter categories to the content view. [#32756]

### Improved compatibility
- Carousel: add new hook allowing third-parties to customize the metadata displayed within the Carousel modal. [#32749]

> - Lazy Images: discourage enabling of Lazy loading images as now 90% of browser support the feature natively and this version in Jetpack can conflict with the upcoming WordPress interactivity API' [#33030]

Reviewer: ```suggestion
- Lazy Images: discourage enabling of Lazy loading images as now 90% of browser support the feature natively and this version in Jetpack can conflict with the upcoming WordPress interactivity API. [#33030]
```

- Subscriptions: change the text color of selected newsletter categories to be more easily readable. [#33104]

> - WC Analytics: ensure correct values are tracked for cart/checkout block/template tracking when using WC Blocks cart and checkout templates [#32715]

Reviewer: ```suggestion
- WC Analytics: ensure correct values are tracked for cart/checkout block/template tracking when using WC Blocks cart and checkout templates. [#32715]
```
